### DISCUSSION TITLE
Welcome to HASS-amazon-rekognition Discussions!

### DISCUSSION THREAD

**Discussion by robmarkcole** [robmarkcole/HASS-amazon-rekognition#75] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by camui-alexandru** [comment]
after latest update, all my template sensors don t work. How can i create a template sensor to show number of cars in picture or number of people?
